spring/boot-autoconfigure/src/main/java/com/linecorp/armeria/internal/spring/ArmeriaConfigurationUtil.java

```diff
@@ -192,6 +169,17 @@ protected HttpResponse doGet(ServiceRequestContext ctx, HttpRequest req)
         }
     }
 
+    private static boolean hasClasses(String... classNames) {
```

Comment:
nit: perhaps `hasAllClasses` is better to understand?

Reply:
Sure. Renamed.